packages/citrix_adc/data_stream/log/_dev/test/pipeline/test-citrix-rfc5424.json

```
@@ -0,0 +1,32 @@
+{
+    "events": [
+        {
+            "@timestamp": "2025-01-12T21:06:37Z",
+            "message": "<131>1 2025-01-12T21:06:37Z MY-CITRIX-HOST ICA 0-PPE-0 - - default Message 4357642 0 :  \"ns_vpn_csg.c:17988 [TECHSUPPORT][LAUNCH][Remote ip = 100.100.100.100:46516] Message = Failed to parse CGP bind request from client\""
```

Could we swap the IP for one that is [in this list](https://github.com/elastic/elastic-package/blob/main/internal/fields/_static/allowed_geo_ips.txt)?
```suggestion
            "message": "<131>1 2025-01-12T21:06:37Z MY-CITRIX-HOST ICA 0-PPE-0 - - default Message 4357642 0 :  \"ns_vpn_csg.c:17988 [TECHSUPPORT][LAUNCH][Remote ip = 175.16.199.1:46516] Message = Failed to parse CGP bind request from client\""
```